proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -220,8 +220,8 @@ message MsgRotateConsPubKey {
   option (gogoproto.goproto_getters) = false;
   option (gogoproto.equal)           = false;
 
-  string              validator_address = 2 [(cosmos_proto.scalar) = "cosmos.ValidatorAddressString"];
-  google.protobuf.Any new_pubkey        = 3 [(cosmos_proto.accepts_interface) = "cosmos.crypto.PubKey"];
+  string              validator_address = 1 [(cosmos_proto.scalar) = "cosmos.ValidatorAddressString"];
```

Comment:
this would be breaking, any chance we can revert it

Reply:
this is never released, this is just added in my last PR here(#18140 )

Reply:
ah okay then ignore me